src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CreateReferralController.java

```diff
@@ -178,6 +179,10 @@ public ResponseEntity<CCDCallbackResponse> aboutToSubmitReferralDetails(
                 userDetails.getName()
             )
         );
+        
+        log.info("Event: Referral Email sent. "
```

Comment:
not sure if it will help in diagnosing the issue of duplicate emails. May be worth checking if a loading/processing can be impmeneted here like RET-3435, in case user clicks the button twice

Reply:
Thanks. I have a look.

Reply:
I had a look and realised that this will change the scope of the ticket. I think from the ticket description, the ticket is intended to help understand what is going wrong and not directly to fix it.

Reply:
Also, the submit button disabling feature that shows response to the user's first click on submit button for the first time, already exists. I tested it on perftest. It does gray out the button + display the "loading" animation.